TeensyDmx.cpp

```diff
@@ -1040,17 +1040,36 @@ void TeensyDmx::startReceive()
 #ifndef IRQ_UART0_ERROR
     if (&m_uart == &Serial1) {
         // Change interrupt vector to mine to monitor RX complete
+        // Fire UART0 receive interrupt immediately after each byte received
+        UART0_RWFIFO = 1;
+
+        // Enable UART0 interrupt on frame error and enable IRQ
         UART0_C3 |= UART_C3_FEIE;
+        NVIC_ENABLE_IRQ(IRQ_UART0_STATUS);
+
         attachInterruptVector(IRQ_UART0_STATUS, UART0RxStatus);
     } else if (&m_uart == &Serial2) {
         // Change interrupt vector to mine to monitor RX complete
+        // Fire UART1 receive interrupt immediately after each byte received
+        UART1_RWFIFO = 1;
+
+        // Enable UART0 interrupt on frame error and enable IRQ
         UART1_C3 |= UART_C3_FEIE;
+        NVIC_ENABLE_IRQ(IRQ_UART1_STATUS);
+
         attachInterruptVector(IRQ_UART1_STATUS, UART1RxStatus);
     } else if (&m_uart == &Serial3) {
         // Change interrupt vector to mine to monitor RX complete
-        UART1_C3 |= UART_C3_FEIE;
+        // Fire UART2 receive interrupt immediately after each byte received
+        UART2_RWFIFO = 1;
+
+        // Enable UART2 interrupt on frame error and enable IRQ
+        UART2_C3 |= UART_C3_FEIE;
+        NVIC_ENABLE_IRQ(IRQ_UART2_STATUS);
+
         attachInterruptVector(IRQ_UART2_STATUS, UART2RxStatus);
     }
+    // TODO: Does this need to handle Serial4-6 here too?
```

Comment:
Does it need doing, or do they only support IRQ_UART0_ERROR method?

Reply:
Did you see this query @chrisstaite ?

Reply:
I saw it, but wasn't sure of the answer myself, therefore happy to leave it as a comment for someone to take on (possibly me) in the future.

Reply:
I've got access to a Teensy 3.5 too now, so when I get a chance, I'll try and test that on all it's serials.

Reply:
Sounds good, happy to add this support later though.